4-Express/client/src/utils/TodoApi.js

```diff
@@ -0,0 +1,66 @@
+const axios = require('axios');
+
+// class TodoApi {
+//     constructor(){
+//         // this.url = url
+//         this.url = "http://localhost:5000/api/todo/"
+//     }
+this.url = "http://localhost:5000/api/todo/"
+
+    const getTodoById = async (todoId) => {
+        try{
+            const res = await axios.get(`${this.url}${todoId}`)
```

Comment:
remember that `await` is blocking the execution so line 12 will be delayed until the http request is resolved.
consider returning the promise and let the consumer decide what to do with it